openmdao/core/petsc_impl.py

```diff
@@ -303,7 +303,7 @@ def setup(self, parent_params_vec, params_dict, srcvec, my_params,
             debug("'%s': creating tgt petsc_vec: (size %d) %s: vec=%s" %
                   (self._sysdata.pathname, len(self.vec), self.keys(), self.vec))
         self.petsc_vec = PETSc.Vec().createWithArray(self.vec, comm=self.comm)
-        if alloc_complex is True:
+        if alloc_complex:
             self.imag_petsc_vec = PETSc.Vec().createWithArray(self.imag_vec,
```

Comment:
nothing significant.  It's really just better form to not check if a boolean value is or is equal to True.  It's mentioned in Pep 8.